### ISSUE TITLE
close button does not work  

### ISSUE THREAD

**Issue by revast** [MizunagiKB/potrace_gui#2]
I found that the close button does not work.
no error message on terminal.

Also, as this is a python3 application, I had to change
`#!/usr/bin/env python`
to
`#!/usr/bin/env python3`

in potrace_gui.py so that it gets started with the right python interpreter when doubleclicking the file.

**Comment by MizunagiKB** [comment]
I'm sorry, because I can not confirm the operation only with macOS, I can not investigate the problem of the close button.
I set the script to be called python3, thanks.

**Comment by MizunagiKB** [comment]
I tried running Potrace GUI 0.2.0 on Ubuntu 16.04.2, but the problem you taught was not reproduced.

![ss_ubuntu](https://user-images.githubusercontent.com/5047878/46247107-7afb2d80-c441-11e8-820a-585a3bf939f3.jpg)


I used [Miniconda](https://conda.io/miniconda.html ) for building Python's environment.

Each version of the main library
- Python 3.6.6 :: Anaconda, Inc.
- PyQt5 5.11.2
- PyQt5_sip 4.19.12
- pillow 5.2.0

Sorry, I am not using Linux, so I will not investigate this issue.

**Comment by revast** [comment]
ok, sorry, maybe I have overseen something. I meant the File-> Close in the menu
![close](https://user-images.githubusercontent.com/8189198/46247208-3606d500-c408-11e8-86c8-013da99d05bc.gif)

**Comment by revast** [comment]
nevertheless, everything else works fine now. And thanks for the new features!

just for reference: 
I have python 3.6.5, pyqt 5.10.1, python3-sip 4.19.7, python-pil (which is a pillow fork) 5.1.0 here. And that is the vanilla xubuntu 18.04 libraries.

**Comment by revast** [comment]
hmm I looked a bit in the code of potrace_gui.py and I see actionFileOpen actionFileSaveAs defined there, but not actionFileClose, in fact there is no "close" string found in the whole file.. as the open and saveAs actions are defined there, I guess its just missing?

**Comment by MizunagiKB** [comment]
OK, I understood. I do not use the menu bar. So I did not notice that the close menu does not work.

It's about this closed menu.
![fixed](https://user-images.githubusercontent.com/5047878/46252581-79635100-c4a5-11e8-991d-0cc0ab325a4c.jpg)

And you are the #3 issue, mentioning application icons.
For this also, I set the application icon.

There have been fixed in [v0.2.1](https://github.com/MizunagiKB/potrace_gui/releases/tag/v0.2.1).
